README.md

```diff
@@ -5,88 +5,112 @@
-send them timely push notifications via FCM.
+send them timely push notifications via [FCM (Firebase Cloud Messaging)](https://firebase.google.com/docs/cloud-messaging).
 
 > ⚠️ **We support Android API level 23 and above** ⚠️
 
 ## Installation
-
-1. Include the [JitPack](https://jitpack.io/#klaviyo/klaviyo-android-sdk) repository in your
-   project's build file
-   ```kotlin
-   // build.gradle.kts
-   allprojects {
-       repositories {
-           maven(url = "https://jitpack.io")
-       }
-   }
-   ```
-   ```groovy
-   // build.gradle
-   allprojects {
-       repositories {
-           maven { url "https://jitpack.io" }
+1. Include the [JitPack](https://jitpack.io/#klaviyo/klaviyo-android-sdk) repository in your project's build file
+   <details>
+      <summary>Kotlin DSL</summary>
+
+      ```kotlin
+      // settings.gradle.kts
+      dependencyResolutionManagement {
+          repositories {
+              maven(url = "https://jitpack.io")
+          }
+      }
+      ```
+   </details>
+
+   <details open>
+      <summary>Groovy</summary>
+
+      ```groovy
+      // settings.gradle
+      dependencyResolutionManagement {
+          repositories {
+              maven { url "https://jitpack.io" }
+          }
+      }
+      ```
+   </details>
+
+2. Add the dependencies to your app's build file
+   <details>
+      <summary>Kotlin DSL</summary>
+
+      ```kotlin
+      // build.gradle.kts
+      dependencies {
+          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0")
+          implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
+      }
+      ```
+   </details>
+
+   <details open>
+      <summary>Groovy</summary>
+
+      ```groovy
+       // build.gradle
+       dependencies {
+           implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
+           implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
        }
-   }
-   ```
-2. Add these dependencies to your app's build file
-   ```kotlin
-   // build.gradle.kts
-   dependencies {
-       implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0")
-       implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
-   }
-   ```
-   ```groovy
-   // build.gradle
-   dependencies {
-       implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
-       implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
-   }
-   ```
-
-## Analytics SDK
-
-### Configuration
-
-The SDK must be configured with the public API key for your Klaviyo account.
-We require access to the `applicationContext` so the SDK can be responsive to
-changes in network conditions and persist data with `SharedPreferences`.
-You must also register the Klaviyo SDK for activity lifecycle callbacks per the example code:
+      ```
+   </details>
+
+## Initialization
+The SDK must be initialized with the 6-character 
```

Comment:
id avoid specifying the length in case it changes

Reply:
NVM I will borrow the language from this section of help center doc: 
https://help.klaviyo.com/hc/en-us/articles/115005062267#difference-between-public-and-private-api-keys1